client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningStyles.ts

```diff
@@ -5,6 +5,10 @@ const useStyles = makeStyles((theme: Theme) => ({
     form: {
         padding: '1vh 1.3vw 2.5vh 1.3vw',
     },
+    occupationSelect: {
```

Comment:
maybe you can show me and we can get this css by the define the grid size

Reply:
👍